### DISCUSSION TITLE
.pt file

### DISCUSSION THREAD

**Discussion by yuhangzhu4** [AdaptiveMotorControlLab/CEBRA#8] [Q&A]
cebra_model.save('/tmp/foo.pt')
What is stored in the .pt file？Can I read specific content？

**Comment by MMathisLab** [comment]
@yuhangzhu4 - this is all based on pytorch, and there is a model load fxn:  `cebra.CEBRA.load("cebra_time_model.pt") `

**Comment by yuhangzhu4** [comment]
Can I know the weight parameters of each feature?
